### PR TITLE
RFC: Update docs

### LINKED ISSUE #3: Info is outdated
The file [build/info.md](https://github.com/coding-in-the-wild/justlogin.xyz/blob/master/build/info.md) is outdated as of `just-login-core` v2.0.0.

### PR DESCRIPTION
**Pull request by ArtskydJ**
Fixes #3.

### CODE REVIEW

**Comment by ArtskydJ** [comment]
Maybe you can review this on Thursday @TehShrike.

**Comment by TehShrike** [comment]
ah yeah!

File: build/info.md [hunk 1]
```diff
@@ -1,57 +1,82 @@
...
+var core = require('just-login-core')(coreDb)
 ```
 
-- Todd receives the email and clicks on the link, which sends his token to the site.
-- Look up the token in the user database.
-- If the token is NOT there, say: "FAILURE! Maybe you waited too long, or clicked an old link."
-- If the token IS there, authenticate the session id associated with the token. (It will, of course, be Todd's record.) Delete the token.
-- Todd has effectively authenticated himself via his email address.
-- No passwords, better security, easier to implement; what's not to like!?
+`core` is an event emitter with functions as properties.
```
Comment (TehShrike) [inline build/info.md]: maybe "core is an event emitter with these functions as properties:"

File: build/info.md [hunk 2]
```diff
@@ -1,57 +1,82 @@
-# Why
+# Documentation
 
-So you're making a site where users need to be authenticated to use its cool service. And you want to use a javascript module to do the authentication stuff.
+Just Login is an password-less authentication system for [node.js][node]. The modules can be installed using [npm][npm], which is shipped with node.js.
 
-Do you really want to have passwords? [Skip passwords!](https://medium.com/@ninjudd/lets-boycott-passwords-680d97eddb01) Think about the advantages:
+Just Login is a modular system. You'll definitely need the `just-login-core`. Want sessions? You'll need `just-login-session-state`. Want emailing? You'll need the `just-login-emailer`.
```
Comment (TehShrike) [inline build/info.md]: These package names should be links

**Comment by TehShrike** [comment]
lpgtm

**Comment by ArtskydJ** [comment]
```js
var client = require('twilio')('ACCOUNT_SID', 'AUTH_TOKEN');

core.on('authentication initiated', function (authReqInfo) {
	var body = 

	client.sendMessage({
		to: authReqInfo.contactAddress,
		from: '+14506667788',
		body: authReqInfo.token
	}, function(err, responseObj) {})
})
```

**Pull request merged**